examples/petsc/Makefile

```diff
@@ -14,6 +14,9 @@ LDLIBS = $(call pkgconf, --libs-only-l $(PETSc.pc) $(ceed.pc)) -lm
 bp.c := $(sort $(wildcard bp*.c))
 bp := $(bp.c:%.c=%)
 
+ns.c := ns.c
```

Comment:
Sorry to keep missing things - The changes in this file are a holdover from when the Navier-Stokes example was in the PETSc folder, right?

Reply:
Thanks for noticing that. Yeah, that was very old